src/main/java/rs/teslaris/core/repository/UserRepository.java

```diff
@@ -0,0 +1,14 @@
+package rs.teslaris.core.repository;
+
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+import rs.teslaris.core.model.User;
+
+@Repository
+public interface UserRepository extends JpaRepository<User, Integer> {
+
+    @Query("select u from User u where u.email = :email")
```

Comment:
This should work without Query annotation? I think the name of finder is in accordance with finders naming convention

Reply:
It should, I just added it because I don't trust the default auto-implementation of queries for some reason 😄 i will remove the annotation.